binding/dotnet/README.md

```diff
@@ -1,104 +1,104 @@
+in Visual Studio or using by using the .NET CLI:
+
 ```console
 dotnet add package PvRecorder
 ```
 
 ## Usage
 
-Getting the list of input devices does not require an instance:
+Initialize and begin recording:
 
 ```csharp
 using Pv;
 
-string[] devices = PvRecorder.GetAudioDevices();
-```
-
-To start recording initialize the instance and run start:
-
-```csharp
-PvRecorder recorder = PvRecorder.Create(
-    deviceIndex: -1, // uses default index
-    frameLength: 512,
-);
-
+PvRecorder recorder = PvRecorder.Create(frameLength: 512);
 recorder.Start();
 ```
 
-Get the pcm frames by calling the read function:
+Read a frame of audio:
 
 ```csharp
 while (true) {
-    pcm = recorder.Read();
-    // do something with pcm
+    short[] frame = recorder.Read();
+    // do something with audio frame
 }
 ```
 
-To stop recording just run stop on the instance:
+To stop recording:
 
 ```csharp
 recorder.Stop();
 ```
 
-Once you are done, free the used resources. You do not have to call stop before Dispose:
+Once you are done, free the used resources. You do not have to call `Stop()` before `Dispose()`:
 
 ```csharp
 recorder.Dispose();
 ```
-
-To have to resources freed immediately after use without explicitly calling the `Dispose` function, wrap PvRecorder in a using statement:
+ 
+To have resources freed immediately after use without explicitly calling the `Dispose()` function, wrap `PvRecorder` in a `using` statement:
 
 ```csharp
-using(PvRecorder recorder = PvRecorder.Create(deviceIndex: -1, frameLength: 512)) {
+using (PvRecorder recorder = PvRecorder.Create(frameLength: 512)) {
     // PvRecorder usage
 }
 ```
 
-### Demo
-
-**NOTE**: The demo is built on .Net Core 3.1.
-
-For more detailed information on how to use the pv_recorder .NET sdk, please that a look at [Demo/Demo.cs](../../demo/dotnet/Demo.cs).
-
-In the following instructions we will refer to  `{AUDIO_DEVICE_INDEX}` as the index of the audio device to use, and `{RAW_OUTPUT_PATH}` as the path to save the raw audio data
-
-`{AUDIO_DEVICE_INDEX}` defaults to -1 and `{RAW_OUTPUT_PATH}` can be empty if you wish to not save any data.
+### Selecting an Audio Device
 
-To build the demo:
-
-```console
-dotnet build -c Release
+To print a list of available audio devices:
+```csharp
+string[] devices = PvRecorder.GetAudioDevices();
 ```
 
-To show the available audio devices run:
-
-```console
-dotnet run --project Demo -c Release -- --show_audio_devices
+The index of the device in the returned list can be used in `Create()` to select that device for audio capture:
+```csharp
+PvRecorder recorder = PvRecorder.Create(
+    frameLength: 512,
+    deviceIndex: 2);
 ```
 
-To run audio recorder run:
 
-```console
-dotnet run --project Demo -c Release
--- --audio_device_index {AUDIO_DEVICE_INDEX} --raw_output_path {RAW_OUTPUT_PATH}
-```
+## Demo
+
+The [PvRecorder .NET demo](https://github.com/Picovoice/pvrecorder/tree/main/demo/dotnet) is a .NET command-line application that demonstrates how to
```

Comment:
I think its better to link it to the relative path

Reply:
This README ends up on NuGet, which is why I use the absolute URL. Otherwise, the link would be broken on NuGet.